addons/docker-swarm.yml

```diff
@@ -0,0 +1,19 @@
+# This playbook has to be copied to the project root directory.
+# See roles/swarm/README.rst for more information.
+
+# CHECK SECURITY - when customizing you should leave this in. If you take it out
+# and forget to specify security.yml, security could be turned off on components
+# in your cluster!
+- include: "{{ playbook_dir }}/playbooks/check-requirements.yml"
```

Comment:
Update on this: It shouldn't.

This role depends on the certificates role, which references the path `ss/cacert.pem`. This path is relative to the working dir of the playbook, which is why the comment at the top of the file is there.